src/Microsoft.OData.Edm/Vocabularies/AlternateKeysVocabularyModel.cs

```diff
@@ -0,0 +1,69 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="AlternateKeysVocabularyModel.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Reflection;
+using System.Xml;
+using Microsoft.OData.Edm.Csdl;
+using Microsoft.OData.Edm.Validation;
+
+
+namespace Microsoft.OData.Edm.Vocabularies.V1
+{
+    /// <summary>
+    /// Representing Core Vocabulary Model.
+    /// </summary>
+    public static class AlternateKeysVocabularyModel
+    {
+        /// <summary>
+        /// The EDM model with core vocabularies.
+        /// </summary>
+        [SuppressMessage("Microsoft.Security", "CA2104:DoNotDeclareReadOnlyMutableReferenceTypes", Justification = "Resolver is immutable")]
+        public static readonly IEdmModel Instance;
+
+        /// <summary>
+        /// The Alternate Keys term.
+        /// </summary>
+        [SuppressMessage("Microsoft.Security", "CA2104:DoNotDeclareReadOnlyMutableReferenceTypes", Justification = "Resolver is immutable")]
+        public static readonly IEdmValueTerm AlternateKeysTerm;
+
+        /// <summary>
+        /// The AlternateKey ComplexType.
+        /// </summary>
+        [SuppressMessage("Microsoft.Security", "CA2104:DoNotDeclareReadOnlyMutableReferenceTypes", Justification = "Resolver is immutable")]
+        public static readonly IEdmComplexType AlternateKeyType;
+
+        /// <summary>
+        /// The PropertyRef ComplexType.
+        /// </summary>
+        [SuppressMessage("Microsoft.Security", "CA2104:DoNotDeclareReadOnlyMutableReferenceTypes", Justification = "Resolver is immutable")]
+        public static readonly IEdmComplexType PropertyRefType;
+
+        /// <summary>
+        /// Parse Alternate Keys Vocabulary Model from CoreVocabularies.xml
+        /// </summary>
+        static AlternateKeysVocabularyModel()

```

Comment:
Should the alternate model be added to EdmModel reference models by default? 

Please refer to Cor/Cap model